paymentapi/pom.xml

```diff
@@ -139,6 +139,18 @@
             <artifactId>spring-boot-devtools</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>order</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>order</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
What does this mean or do?

Reply:
I need to check if it came from my first tries to use orderApi controllers/services from paymentApi to create orders and refunds. I'll remove it and retest without

Reply:
Removed